### PR TITLE
Alkwa/fix release cleanup

### PR DESCRIPTION
**Pull request by alkwa-msft**
# What
There was an issue with referencing the clean up script. I also wanted to clean up the name of the script to align with the other scripts. Finally we also included the script in an unintended area so we are cleaning that up

# Why
It was breaking Ci and not allowing us to deploy releases.

# How Tested
Tested building out an alpha release

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: rush.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.109.1",
```
Comment (JamesBurnside) [inline rush.json:3]: revert change to rush.json, this is being handled in its own PR #3700

File: .github/workflows/alpha-release.yml [hunk 2]
```diff
@@ -84,8 +84,7 @@ jobs:
 
       # Clean package.json
       - name: Remove scripts, devDependencies, and beachball from package.json
-        run: node ./common/scripts/preparePackageJSON.js
-        working-directory: ./packages/communication-react
```
Comment (JamesBurnside) [inline .github/workflows/alpha-release.yml:88]: Without this, how does it know to change the package.json in communication-react?

image

File: rush.json (hunk 1 above)
Comment (alkwa-msft) [reply rush.json:3]: right i was going to remove this

File: .github/workflows/alpha-release.yml (hunk 2 above)
Comment (alkwa-msft) [reply .github/workflows/alpha-release.yml:88]: ill need to look further into this

File: rush.json (hunk 1 above)
Comment (emlynmac) [reply rush.json:3]: Looks like the issue may not be the rush update after all.
There's an issue with `rush switch-flavor:beta-release` and it looks like the only real difference between 109 and previous rush versions is that the error is now reported differently.

File: .github/workflows/alpha-release.yml (hunk 2 above)
Comment (alkwa-msft) [reply .github/workflows/alpha-release.yml:88]: Should be fixed now

File: rush.json (hunk 1 above)
Comment (alkwa-msft) [reply rush.json]: I have reverted the 5.109 change so we should be good here

**Review by JamesBurnside — APPROVED** [review]

**Review by emlynmac — APPROVED** [review]

**Pull request merged**